components/icons/DisabledIcons.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <v-row justify="space-around">
+        <v-icon
+          large
+          color="green darken-2"
+          disabled
+        >
+          mdi-domain
+        </v-icon>
+
+        <v-icon aria-label="My Account" role="img""
+        disabled>
+          mdiAccount
+        </v-icon>
+
+        <script>
```

Comment:
Please reformat code like that template:
< template> .... </template>
< script> ... </script>